### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Nahid1911**
### **Requirements:**

1. Use Gitflow.
2. Create an npm project with webpack.
3. Write plain HTML markup with minimum styling.
4. Use ES6 modules, with import and export.

**For more requirements please click [HERE](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md).** 🔗

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable*/
```
Comment (Strangeal) [inline src/index.js]: - [ ] Kindly consider avoiding the disabling of `ESLint` rules as it is designed to assist in writing code that aligns with `industry standards` and `best practices`. Disabling `ESLint` rules can lead to code that may not adhere to these standards, potentially resulting in code quality and maintainability issues. Instead, it is recommended to address any specific `ESLint` rule violations by finding alternative solutions, ensuring that the code remains consistent and of high quality.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Nahid1911, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Nahid1911) [reply src/index.js]: @Strangeal 
resolved the issue

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Nahid1911, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**